examples/addweighted.js

```diff
@@ -1,14 +1,14 @@
 var cv = require('../lib/opencv');
 
-cv.readImage("./files/mona.png", function(err, orig) {
+cv.readImage("./examples/files/mona.png", function(err, orig) {
```

Comment:
I suppose the path behaviour is a little strange.

Everything is called as a child process (see test/examples.js). So we don't need try/catches.

I'll change the CWD